client/src/components/Greeting.js

```diff
@@ -0,0 +1,25 @@
+import "./Greeting.css";
+
+export default function Greeting({ user, isUserLoggedIn }) {
+  const greetings = [
+    "what's cookin'?",
+    "what's happening?",
+    "what's the plan?",
+    "let's get preppin'?",
+    "what's on your remind?",
+    "got a new idea?",
+    "what's lined up?",
+  ];
+
+  const number = Math.floor(Math.random() * greetings.length);
+  return (
+    <h1 className='greeting'>

```

Comment:
This is just depending on which convention you want to use here. But normally i'll name the class of the root element of a component with the same name of the component (in this case, `"Greeting"`). Same for the other components